test/e2e/tkn_helper_utility.go

```diff
@@ -0,0 +1,316 @@
+package e2e
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"gotest.tools/icmd"
+	knativetest "knative.dev/pkg/test"
+
+	"knative.dev/pkg/test/logging"
+
+	//"github.com/tektoncd/cli/test/helper"
+	"github.com/tektoncd/pipeline/pkg/names"
+	"golang.org/x/xerrors"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	// Mysteriously by k8s libs, or they fail to create `KubeClient`s from config. Apparently just importing it is enough. @_@ side effects @_@. https://github.com/kubernetes/client-go/issues/242
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+)
+
+var initMetrics sync.Once
+
+const (
+	interval = 1 * time.Second
+	timeout  = 10 * time.Minute
+)
+
+func Setup(t *testing.T) (*Clients, string) {
```

Comment:
Same as above, there might be some work to be done upstream to not to have to duplicate this in here.